stacker/lookups/handlers/output.py

```diff
@@ -5,48 +5,25 @@
 Output = namedtuple("Output", ("stack_name", "output_name"))
 
 
-def handler(value, provider=None, context=None, fqn=False, rfqn=False,
-            **kwargs):
+def handler(value, context=None, **kwargs):
     """Fetch an output from the designated stack.
 
     Args:
         value (str): string with the following format:
             <stack_name>::<output_name>, ie. some-stack::SomeOutput
-        provider (:class:`stacker.provider.base.BaseProvider`): subclass of the
-            base provider
         context (:class:`stacker.context.Context`): stacker context
-        fqn (bool): boolean for whether or not the
-            :class:`stacker.context.Context` should resolve the `fqn` of the
-            stack.
-        rfqn (bool): boolean for whether or not the
-            :class:`stacker.context.Context` should resolve the `fqn` of the
-            stack prefixed by the namespace variable
 
     Returns:
         str: output from the specified stack
 
     """
 
-    if rfqn:
-            value = "%s%s%s" % (
-                    context.namespace,
-                    context.namespace_delimiter,
-                    value
-            )
-
-    if provider is None:
-        raise ValueError('Provider is required')
     if context is None:
         raise ValueError('Context is required')
 
     d = deconstruct(value)
-
-    stack_fqn = d.stack_name
-    if not fqn:
-        stack_fqn = context.get_fqn(d.stack_name)
-
-    output = provider.get_output(stack_fqn, d.output_name)
-    return output
+    stack = context.get_stack(d.stack_name)
```

Comment:
Because I figure this question will get asked, "What happens if `get_stack` returns None, or outputs haven't been set?":

The graph ensures that that scenario can't happen (remember, the graph is being built out based on the `output` lookups). If for some reason it _did_ happen, then the step would just blow up, and any downstream steps would be cancelled.